dotCMS/src/main/java/com/dotcms/rest/api/v1/apps/AppsResource.java

```diff
@@ -110,10 +110,10 @@ public final Response listAvailableServices(@Context final HttpServletRequest re
     @JSONP
     @NoCache
     @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
-    public final Response getServiceIntegrationByKey(
+    public final Response getAppByKey(
             @Context final HttpServletRequest request,
             @Context final HttpServletResponse response,
-            @PathParam("key") final String serviceKey,
+            @PathParam("key") final String key,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "key" appears 4 times in this file; the first occurrence is on line 116](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5218098)